nuxt.config.js

```diff
@@ -132,6 +135,11 @@ module.exports = {
         silentFallbackWarn: true,
         dateTimeFormats: i18nDateTime
       },
+      //disable redirects to callback page or authentication fails
+      parsePages: false,
```

Comment:
This is to prevent generating localized routes for "callback" page. It will only affect whatever is in the "pages" option underneath. From the docs:
"If you disabled parsePages option, localization can be disabled for specific pages and locales by setting the unwanted locale(s). To completely disable routes localization on a given page set to false".